src/modules/dpos/validator/components/Overview/delegatesOverview.js

```diff
@@ -52,31 +53,39 @@ const Overview = ({
     },
   };
 
+  const totalDelegates = () => (
+    <>
+      <p>1804</p>
+      <span>Total delegates</span>
+    </>
+  );
```

Comment:
@eniolam1000752 This might need to be made dynamic or completely removed because it affects the display of the chart details on hover